api/src/main/java/org/openmrs/module/ugandaemrreports/definition/dataset/definition/EarlyWarningIndicatorsDatasetDefinition.java

```diff
@@ -0,0 +1,43 @@
+package org.openmrs.module.ugandaemrreports.definition.dataset.definition;
+
+import org.openmrs.module.reporting.dataset.definition.BaseDataSetDefinition;
+import org.openmrs.module.reporting.definition.configuration.ConfigurationProperty;
+
+import java.util.Date;
+
+/**
+ * Created by carapai on 06/04/2016.
+ */
+public class EarlyWarningIndicatorsDatasetDefinition extends BaseDataSetDefinition {
```

Comment:
@carapai Can we have a UgandaEMRDataSet definition from which others inherit so that we do not have to duplicate this code - I think I may have seen it elsewhere too

Reply:
We will do that